README.md

```diff
@@ -49,6 +49,7 @@ We rely on different `django` and `mypy` versions:
 
 | django-stubs   | Mypy version | Django version | Django partial support | Python version |
 |----------------|--------------|----------------|------------------------|----------------|
+| (next release) | 1.7.x        | 5.0            | 4.2, 4.1, 3.2          | 3.8 - 3.12     |
```

Comment:
I think that `3.8 - 3.12` is not correct for `Django@5.0`

Reply:
Sure, but that's Django's constraint, not ours. `django-stubs` itself still supports Python 3.8 with Django 4.2.

Reply:
Ok 👍

Reply:
Should I count this as approval to merge?